tests/Aspire.CommunityToolkit.Hosting.Meilisearch/AppHostTests.cs

```diff
@@ -0,0 +1,41 @@
+using Aspire.CommunityToolkit.Testing;
+using Aspire.Components.Common.Tests;
+using FluentAssertions;
+using System.Net.Http.Json;
+
+namespace Aspire.CommunityToolkit.Hosting.Meilisearch.Tests;
+
+[RequiresDocker]
+public class AppHostTests(AspireIntegrationTestFixture<Projects.Aspire_CommunityToolkit_Hosting_Meilisearch_AppHost> fixture) : IClassFixture<AspireIntegrationTestFixture<Projects.Aspire_CommunityToolkit_Hosting_Meilisearch_AppHost>>
+{
+    [Theory]
+    [InlineData("meilisearch")]
+    public async Task ResourceStartsAndRespondsOk(string resourceName)
+    {
```

Comment:
Only need to use a `Fact` here as there's a single resource we're testing against.

```suggestion
    [Fact]
    public async Task ResourceStartsAndRespondsOk()
    {
        var resourceName = "meilisearch";
```